src/main/java/sentinel/command/FindCommand.java

```diff
@@ -0,0 +1,17 @@
+package sentinel.command;
+
+import sentinel.ui.Ui;
+import sentinel.utils.Parser;
+import sentinel.utils.SentinelList;
+
+public class FindCommand extends Command{
+    public FindCommand(Ui ui, SentinelList list){
+        super(ui, list);
+    }
+
+    @Override
+    public void execute(String input) {
+        String keyword = Parser.parseKeyword(input);
+        ui.showFileredList(list, keyword);
```

Comment:
Slight typo for "showFileredList", should be showFilteredList 👍

Reply:
Must have missed it doing it blurry-eyed in the middle of the night, haha! Thanks!